packages/core/src/2d/sprite/SpriteMask.ts

```diff
@@ -208,6 +208,29 @@ export class SpriteMask extends Renderer {
     BatchUtils.batchFor2D(elementA, elementB);
   }
 
+  /**
+   * @internal
+   */
+  override _onEnableInScene(): void {
+    super._onEnableInScene();
+    this._renderElement = new RenderElement();
```

Comment:
- `RenderElement` and `SubRenderElement` Can new  when create 
- This func should do ` engine._maskManager.addSpriteMask(this);`

Reply:
`This func should do engine._maskManager.addSpriteMask(this);` 
We can not do this, because mask belong to scene.